src/nextjournal/clerk/config.clj

```diff
@@ -1,37 +1,54 @@
 (ns nextjournal.clerk.config
-  (:require [clojure.edn :as edn]
-            [clojure.java.io :as io]
-            [clojure.string :as str]))
+  (:require
+   [babashka.fs :as fs]
+   [clojure.edn :as edn]
+   [clojure.java.io :as io]
+   [clojure.string :as str]))
 
 (def cache-dir
   (or (System/getProperty "clerk.cache_dir")
-      ".clerk/cache"))
+      ".clerk/.cache"))
 
 (def cache-disabled?
   (when-let [prop (System/getProperty "clerk.disable_cache")]
     (not= "false" prop)))
 
 (def gs-url-prefix "https://storage.googleapis.com/nextjournal-cas-eu")
-(def lookup-hash (str/trim (slurp (io/resource "viewer-js-hash"))))
-(def lookup-url (str gs-url-prefix "/lookup/" lookup-hash))
+(def viewer-lookup-hash (str/trim (slurp (io/resource "viewer-js-hash"))))
+(def viewer-lookup-url (str gs-url-prefix "/lookup/" viewer-lookup-hash))
+(def viewer-local-lookup (str (io/file cache-dir "lookup" viewer-lookup-hash)))
 
 (def resource-manifest-from-props
   (when-let [prop (System/getProperty "clerk.resource_manifest")]
     (when-not (str/blank? prop)
       (read-string prop))))
 
+(def cached-viewer-lookup
+  "Lookup of viewer js. Loaded from local file system after caching."
+  (delay
+    (edn/read-string
+     (if (fs/exists? viewer-local-lookup)
+       (slurp viewer-local-lookup)
+       (let [lookup (slurp viewer-lookup-url)]
+         (fs/create-dirs (fs/parent viewer-local-lookup))
+         (spit viewer-local-lookup lookup)
+         lookup)))))
```

Comment:
Wondering if we can make things a bit more explicit here. Something to talk about on our call.